src/Castle.Windsor.Extensions.DependencyInjection.Tests/Castle.Windsor.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;netcoreapp3.0;netcoreapp3.1;net461;net471;net48</TargetFrameworks>
```

Comment:
What is this for? Why do we need to compile the test project on all these platforms, especially with .NET Framework 4.x being an in place upgrade so a machine can only have a single version. If you want to run the unit tests on multiple .NET Core versions that'll be handled through the AppVeyor config in the root of the repo and the build script.

Reply:
The AppVeyor build is also failing because it doesn't have .NET Core 3.0 installed, maybe need to pick a different AppVeyor image?

Reply:
> What is this for? Why do we need to compile the test project on all these platforms, especially with .NET Framework 4.x being an in place upgrade so a machine can only have a single version. If you want to run the unit tests on multiple .NET Core versions that'll be handled through the AppVeyor config in the root of the repo and the build script.

Well before it was testing net461 and netcoreapp2.2. I can revert. This is how i normally Do multiple framework testing. It works very Well.
In this case, my thought was, "the issue was specific to netcoreapp3.1, then we should also run the tests for netcoreapp3.1"
If you have a better way to do it, I can revert to net461 and netcoreapp2.2 again?

Reply:
It appears you are right that this is the way it is done so `dotnet test` and Visual Studio work, even though the test project get recompiled for each target. Go for it with the .NET Core targets, but I don't see any reason to add more .NET Framework targets since they are all the same runtime.

You'll likely have to update the AppVeyor config YAML file to get newer .NET Core runtimes on the build machine.

Reply:
@jonorossi I don't see any references to the netcore versions available in the appveyor.yml I can change to the VS2019 image, but that is all. Only you have login to AppVeyor?

Reply:
VS2019 AppVeyor image is the right fix, AppVeyor has the SDKs installed on the images.